docs/cluster-agent/HORIZONTAL_POD_AUTOSCALING.md

```diff
@@ -0,0 +1,292 @@
+# Datadog Cluster Agent - DCA | Getting Started
+
+The DCA is a **beta** feature, if you are facing any issues please reach out to our [support team](http://docs.datadoghq.com/help)
+
+## Introduction
+
+The Horizontal Pod Autoscaling feature has been introduced in [Kubernetes v1.2](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/#before-you-begin).
+It allowed users to autoscale off of basic metrics like CPU, but required a resource called the metrics-server to run along side your application.
+As of Kubernetes v1.6, it is possible to autoscale off of [custom metrics](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#support-for-custom-metrics).
+Custom metrics are user defined and are collected from within the cluster.
+As of Kubernetes v1.10, the External Metrics support was introduced. Users are now able to autoscale off of any metric not restricted to the cluster.
```

Comment:
"...support for external metrics was introduced so users can autoscale off of any metric from outside the cluster that is collected for you by Datadog."